Examples/Python/tests/test_examples.py

```diff
@@ -923,7 +922,7 @@ def test_ckf_tracks_example_truth_smeared(tmp_path, assert_root_hash):
 
 @pytest.mark.skipif(not dd4hepEnabled, reason="DD4hep not set up")
 @pytest.mark.slow
-# @pytest.mark.filterwarnings("ignore::UserWarning")
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
@paulgessinger, why was this line commented out? It gives a warning in pytest. (The warning changed a bit with this PR, but it's still there.)
Another way to fix this would be to disable ROOT output, since the output is not checked. However it's perhaps useful to check that the output doesn't crash.

Reply:
I don't remember. I think this will discard the warning though. Do we want to display it?

Reply:
So that means we should remove this line altogether, right?

Reply:
This line supresses the expected warning in the pytest. Isn't that what we want?

Reply:
Should be alright.